src/default-retrieve-file-handler.ts

```diff
@@ -1,23 +0,0 @@
-import * as fs from 'fs';
-import { transpileIfTypescript } from './transpile-if-ts';
-
-export function defaultRetrieveFileHandler(path) {
-  // Trim the path to make sure there is no extra whitespace.
-  path = path.trim();
-
-  // This was removed because it seems that we can't use cache while expecting correct results
-  // TODO: check correctness and performance with file caching 
-  // if (path in fileContentsCache) {
-  //   return fileContentsCache[path];
-  // }
-
-  var contents: string;
-  try {
-    contents = fs.readFileSync(path, 'utf8');
-    contents = transpileIfTypescript(path, contents);
```

Comment:
Ok, I understand. But we have tests that checks correctness of error mapping, even in watch mode. Plus after review of [source-map-support](https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js) I'm pretty sure in what cases this function is called and in that cases we actually don't need extra transpilation, because it used:

1. [Here](https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js#L112) for reading js-file and searching for source-map url.
2. [Here](https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js#L143) for reading ts-file, but source-map that point to it already exists, and file has no need to be transpiled.